README.md

```diff
@@ -34,6 +34,11 @@ See the [Fauna Documentation](https://docs.fauna.com/fauna/current/) for additio
     - [Iterate on a stream](#iterate-on-a-stream)
     - [Close a stream](#close-a-stream)
     - [Stream options](#stream-options)
+  - [Event Feeds](#event-feeds)
+    - [Request an Event Feed](#request-a-event-feed)
+    - [Iterate on an Event Feed](#iterate-on-a-event-feed)
+    - [Error handling](#error-handling)
+    - [Event Feed options](#event-feed-options)
```

Comment:
Can we move this above the Event Streaming section? Bryan requested that we give Event feeds top billing.